README.md

```diff
@@ -247,9 +247,11 @@ target create "/Users/yourname/electron-gn/src/out/Testing/Electron.app/Contents
 
 ### `e test`
 
-`e test ` starts the local Electron build's test runner. Any extra args are passed
+`e test ` starts the local Electron build's test runner by default. Any extra args are passed
 along to the runner.
 
+You can also optionally specify an Electron executable to run by passing the path to a directory containing an Electron executable, as well as set a configuration option allowing you to specify a directory containing multiple Electron executable files by setting `config.electronBinaryDirectory` and then passing `--v=x.y.z`.
```

Comment:
Presumably this means manually modifying the config under ~/.electron_build_tools/configs/ to include that setting? That's not quite clear since there's no other mention of manually modifying configs (or where they are) in README.md and no commands for modifying configs.

Reply:
Yeah i thought about this a bit and tbh am not sure - if you were reading this, what do you think would make this clear to you? Or - what dx would you want to see?

Reply:
I think there should either be a way to modify the configs built into `e` (obviously that's a bit of work outside the scope of this), or a section of README.md which mentions where configs are stored for modifying them manually. Then this blurb could say: "by setting `config.electronBinaryDirectory` (see \[Modifying Configs](#modifying-configs))" or something like that.

Just my two cents. In the current form that's only going to be clear to people who have already poked around a bunch in `build-tools` since `config.electronBinaryDirectory` is pretty opaque without that knowledge.

Reply:
Not a blocker for this PR, but what I had in mind was something like `e start --fiddle=vX.Y.Z` and build-tools would Just Work by finding Fiddle's appData path automatically. This is my preference because then the script will do the work I'd be doing by hand on win/mac/linux.

This PR's mechanism is more universal, e.g. it doesn't require people to be using Fiddle in order to work, and I like that. IMO `--fiddle` could work by reusing some of this PR's code and would be very convenient for the use case of fiddle + buld-tools.

Adding `--fiddle` would require us to clone Electron's per-platform appData logic -- or more likely, to import it e.g from Sindre's env-paths package. I'd still be :+1: on this either in this PR if you're so inclined, or for any of us to write it in a followup PR.